src/index.js

```diff
@@ -729,7 +729,14 @@ export default class HyperScreen extends React.Component {
           formData.append(name, input.getAttribute('value'));
         }
       });
-    return formData;
+
+    let formHasData = false;
+    for (var key of formData.keys()) {
```

Comment:
Couldn't find a quick way to see if a `FormData` is empty. Consequently, had to rely on the good 'ole for-loop. Happy to accept suggestions for how to do this non-imperatively

Reply:
Would this work?
```js
const hasFormData = formData.entries().length > 0;
```

Reply:
@flochtililoch tried it and didn't work. It appears ([from docs I found](
https://developer.mozilla.org/en-US/docs/Web/API/FormData)) that although `FormData` has helpers that return an iterator, there's no quick way to grab length.

<img width="716" alt="screen shot 2019-03-02 at 7 21 17 pm" src="https://user-images.githubusercontent.com/15131271/53690499-612b8280-3d20-11e9-8f45-b68fa5f95a95.png">

Reply:
After doing some experimenting in the "Basic forms" case study of the demo app, it does not appear that `FormData` instances have `keys` or `entries` methods. The object looks like:
```json
{"_parts":[["data", "Test"]]}
```
(`Test` being the value typed in the text input field)

Reply:
Is that part of the internal implementation though? Might not be a good idea to rely on the internals, which may change in the future.

Reply:
@adamstep I checked for properties of `FormData` (by printing `Object.getOwnPropertyNames(formData)`), and there doesn't seem to be other methods or properties other than `_parts`

Reply:
I settled on another way that doesn't involve having to inspect `formData`